test/integration/fetchcontent_compatibility.rb

```diff
@@ -0,0 +1,44 @@
+require_relative './lib'
```

Comment:
file should be prefixed with `test_` in order to be automatically run by the framework: ie `test_fetchcontent_compatibility.rb`